api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * This file is part of adventure, licensed under the MIT License.
+ *
+ * Copyright (c) 2017-2020 KyoriPowered
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package net.kyori.adventure.audience;
+
+import net.kyori.adventure.bossbar.BossBar;
+import net.kyori.adventure.sound.Sound;
+import net.kyori.adventure.sound.SoundStop;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.title.Title;
+import org.checkerframework.checker.nullness.qual.NonNull;
+
+/**
+ * An audience.
+ */
+public interface Audience {
+  /**
+   * Sends a message.
+   *
+   * @param message the message
+   */
+  void message(final @NonNull Component message);
+
+  /**
+   * Shows a bossbar.
+   *
+   * @param bar the bossbar
+   */
+  void showBossBar(final @NonNull BossBar bar);
+
+  /**
+   * Hides a bossbar.
+   *
+   * @param bar the bossbar
+   */
+  void hideBossBar(final @NonNull BossBar bar);
+
+  /**
+   * Shows a message on the action bar.
+   *
+   * @param message the message
+   */
+  void showActionBar(final @NonNull Component message);
+
+  /**
+   * Plays a sound.
+   *
+   * @param sound the sound
+   */
+  void playSound(final @NonNull Sound sound);
+
+  /**
+   * Stops all sounds.
+   *
+   * @param stop the stop
+   */
+  void stopSound(final @NonNull SoundStop stop);
+
+  /**
+   * Shows a title.
+   *
+   * @param title the title
+   */
+  void showTitle(final @NonNull Title title);
+
+  /**
+   * Clears the title.
```

Comment:
Would be nice to have a brief blurb on the difference between clear and reset. TBH, I don't even know the diff.

Reply:
https://github.com/KyoriPowered/text/pull/62/commits/fd7ce1a567b36ac5cda35207c90354e398777f5d